src/webtransport.js

```diff
@@ -241,7 +319,9 @@ class Http3WTStream {
           const res = this.pendingres
           this.pendingoperation = null
           this.pendingres = null
-          res()
```

Comment:
Should error, because it indicates a bug in the implementation

Reply:
Or no `this.pendingoperation` guards this already, does `tsc` require an explicit type checking?

Reply:
`tsc` requires explicit type checking - if the value can be `null` it needs be handled as such rather than just letting it fail with a runtime error.

Reply:
ok, but it should never be null, only if there is an error in the code. but ok